android/app/src/main/res/values/strings.xml

```diff
@@ -219,7 +219,10 @@
     contacting certain domains or websites known for distributing ads, malware, trackers and
     more.</string>
     <string name="dns_content_blockers_warning">This might cause issues on certain websites,
-    services, and programs.</string>
+    services, and apps.</string>
+    <string name="dns_content_blockers_custom_dns_warning">
+<![CDATA[Attention: this setting cannot be used in combination with <b>Use custom DNS server.</b>]]>
```

Comment:
I don't think the full stop should be in bold.

```suggestion
<![CDATA[Attention: this setting cannot be used in combination with <b>Use custom DNS server</b>.]]>
```

Reply:
Thank you, fixed. :)